pallets/salp/src/lib.rs

```diff
@@ -262,16 +381,19 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			#[pallet::compact] index: ParaId,
 			#[pallet::compact] cap: BalanceOf<T>,
-			#[pallet::compact] first_slot: LeasePeriodOf<T>,
-			#[pallet::compact] last_slot: LeasePeriodOf<T>,
-			#[pallet::compact] end: T::BlockNumber,
+			#[pallet::compact] first_slot: LeasePeriod,
+			#[pallet::compact] last_slot: LeasePeriod,
 		) -> DispatchResultWithPostInfo {
 			let depositor = ensure_signed(origin)?;
 
 			ensure!(first_slot <= last_slot, Error::<T>::LastSlotBeforeFirstSlot);
-			let last_slot_limit = first_slot.checked_add(&3u32.into()).ok_or(Error::<T>::FirstSlotTooFarInFuture)?;
-			ensure!(last_slot <= last_slot_limit, Error::<T>::LastSlotTooFarInFuture);
-			ensure!(end > frame_system::Pallet::<T>::block_number(), Error::<T>::CannotEndInPast);
+			let last_slot_limit = first_slot
+				.checked_add(7u32.into())
```

Comment:
seems slot range is 8 in polkadot, need some review and confirm

Reply:
total slot range is 8, include the first slot